### PR TITLE
mvc-2

### PR DESCRIPTION
**Pull request by Rommelua**

### CODE REVIEW

**Review by andromash — APPROVED** [review]
Good job!

File: src/main/java/com/dev/cinema/controller/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    @PostMapping("/complete")
+    public void completeOrder(@RequestParam Long userId) {
+        ShoppingCart shoppingCart = cartService.getByUser(new User(userId));
+        orderService.completeOrder(shoppingCart);
+    }
+
+    @GetMapping
+    public List<OrderResponseDto> getOrderHistory(@RequestParam Long userId) {
+        return orderService.getOrderHistory(new User(userId)).stream()
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/controller/OrderController.java:45]:
```suggestion
        return orderService.getOrderHistory(new User(userId)).stream()
```
I know that you only need userId to find his orders, but let's pass the whole user to this method

File: src/main/java/com/dev/cinema/controller/ShoppingCartController.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        this.cartService = cartService;
+        this.sessionService = sessionService;
+        this.cartMapper = cartMapper;
+    }
+
+    @PostMapping("/movie-sessions")
+    public void addMovieSessionToCart(@RequestParam Long sessionId,
+                                      @RequestParam Long userId) {
+        MovieSession movieSession = sessionService.get(sessionId);
+        User user = new User(userId);
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/controller/ShoppingCartController.java]: same

File: src/main/java/com/dev/cinema/controller/OrderController.java (hunk 1 above)
Comment (Rommelua) [reply src/main/java/com/dev/cinema/controller/OrderController.java:45]: done

File: src/main/java/com/dev/cinema/controller/ShoppingCartController.java (hunk 2 above)
Comment (Rommelua) [reply src/main/java/com/dev/cinema/controller/ShoppingCartController.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**